packages/suite-desktop/scripts/build.js

```diff
@@ -44,12 +44,17 @@ console.log('[Electron Build] Starting...');
 console.log(`[Electron Build] Mode: ${isDev ? 'development' : 'production'}`);
 console.log(`[Electron Build] Using mocks: ${useMocks}`);
 
+// we want to bundle this
```

Comment:
This is already solved in https://github.com/trezor/trezor-suite/pull/4797 by @mroz22